src/app/app.component.html

```diff
@@ -7,25 +7,30 @@
 
   <div class="top-overlay action-icon-container">
     <label class="no-margin">
-      <fa-icon class="action-icon"
+      <fa-icon class="action-icon" data-tooltip="Import File"
```

Comment:
Think I used `data-ttu-` as prefix somewhere in the code, so I would prefer if it follows that convention (sorry for not documenting it)